data/json/snippets/snippets.json

```diff
@@ -43,7 +43,10 @@
     "type": "snippet",
     "category": "charsheet",
     "text": [
-      { "id": "charsheet_1", "text": "It's a Dungeons & Dragons 6th Edition character sheet.  This one is for a fighter." },
```

Comment:
This is dictated by our JSON dialect.  Changing from line 46 would permanently break our json validation tests.

Reply:
Hrm. I apologize for that; my rationale was that I was standardizing with the lines below it.

Reply:
Uhh, how do I accept this change? Last time there was a button for it. Do I have to manually commit this time? Sorry, relatively new.

Reply:
https://dev.narc.ro/cataclysm/format.html

Yeah those two lines look very similar so they are probably a few characters off from each other and one tripped the character limit and the other didn't.  No problem, once you run your files through the linter and copy them back into your editor then reupload them to github I'll rereview.

Reply:
OK, I've re-pushed the files that failed linting. What's really weird is that some lines are fine, and some aren't. You said there's a character limitation per line that causes linting to fail? I'll have to bear this linter in mind going forward; thanks!